base/Dockerfile

```diff
@@ -19,6 +21,15 @@ RUN apt update -y && \
         wget \
         ca-certificates
 
+RUN TAR_PACKAGES="http://gca-jobs:1234/packages/libpimega_2.5.1-0_amd64.tar.gz" && \
```

Comment:
The libpimega version should be a variable.

Reply:
`NDSSCPimega` should also follow this logic, right?

Reply:
Yup